docs/design/host_to_services_OpenFlow.md

```diff
@@ -7,11 +7,42 @@ In order to allow host to Kubernetes service access packets originating from the
 
 ## Introduction
 
-The DGP solution functionally works, but adds complexity to the OVN-Kubernetes network architecture and creates an additional path into the host which is undesirable for Smart NIC deployments. A new implementation, leverages OpenFlow on the shared gateway bridge to handle all host to service traffic, and eliminates the need for DGP. This feature is only supported with shared gateway mode.
+The DGP solution functionally works, but adds complexity to the OVN-Kubernetes network architecture and creates an 
+additional path into the host which is undesirable for Smart NIC deployments. A new implementation, leverages OpenFlow 
+on the OVS gateway bridge to handle all host to service traffic, and eliminates the need for DGP. 
+This feature is only supported with shared and local gateway modes.
+
```

Comment:
isn't DGP no longer supported i.e history ? if so why we still referring to it why not just remove any reference to DGP mode ?

Reply:
ok will do